.gitignore

```diff
@@ -1,2 +1,3 @@
 */test/.helm
-*/test/.git-sources
\ No newline at end of file
+*/test/.git-sources
+*/test/.tilt-cache
```

Comment:
re: your bigger question about tilt caching -

ya, "fetch this remote tarball and cache the result" is turning out to be a pretty common pattern. 

I like your point about whether it should be a per-repo cache or a cache in $HOME/.cache.

I feel like most build systems I've worked with end up making some sort of a primitive for this to make this easy. But this seems fine for now.